### PR TITLE
Make coding-standard robust against execution from other paths

### PR DESCRIPTION
**Pull request by rdss-sknott**
* Ask the surrounding git repo for it's root
* Distinguish between root directory of code to check and directory
  where coding-standard is installed

# How to check that it works?

* Clone Coding-Standard to filesystem (e.g. /a/cs)
* Composer install
* Switch to another non empty directory outside of coding-standard (e.g. /b)
* Call coding-standard from here (e.g. ../a/src/bin/coding-standard sca:all)
* All PHP-Checks should work as expected

### CODE REVIEW

File: src/main/php/CommandLine/Library/Environment.php [hunk 1]
```diff
@@ -45,8 +45,18 @@ public function __construct(
      */
     public function getRootDirectory() : string
     {
-        $projectRootPath = ComposerLocator::getRootPath();
-        return realpath($projectRootPath);
+        $projectRootPath = $this->processRunner->runAsProcess('git', 'rev-parse', '--show-toplevel');
+        return $projectRootPath;
```
Comment (rdss-pwacht) [inline src/main/php/CommandLine/Library/Environment.php:49]: Maybe it make sense to check bevor, if the directory have .git-file?

**Review by rdss-pwacht — APPROVED** [review]
Best Change ❤️  

✅ Test pass:

```
patrizia.wacht@digital1560 weinkeller % ../coding-standard/src/bin/coding-standard sca:all 
All SCA-Commands will be executed.
PHPPL : Running full check
PHPCS : Running full check

FILE: /Users/patrizia.wacht/workspace/weinkeller/autoload.php
----------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 3 LINES
----------------------------------------------------------------------
 26 | ERROR | [x] Use statements should be sorted alphabetically. The
    |       |     first wrong one is
    |       |     Composer\Autoload\ClassLoader.
    |       |     (SlevomatCodingStandard.Namespaces.AlphabeticallySortedUses.IncorrectlyOrderedUses)
 28 | ERROR | [ ] Missing short description in doc comment
    |       |     (ZooRoyal.Commenting.DocComment.MissingShort)

```

**Comment by rdss-pwacht** [comment]
Perfekt 
Bildschirmfoto 2021-02-04 um 1 10 14 PM

**Review by rdss-pwacht — APPROVED** [review]

**Review by rdss-rsanthanam — APPROVED** [review]
awesome 👍🏼

**Pull request merged**